services/director-v2/src/simcore_service_director_v2/api/routes/computations.py

```diff
@@ -98,20 +98,20 @@
 router = APIRouter()
 
 
-async def _check_pipeline_not_running(
+async def _check_pipeline_not_running_or_raise_409(
     comp_tasks_repo: CompTasksRepository, computation: ComputationCreate
 ) -> None:
     pipeline_state = utils.get_pipeline_state_from_task_states(
         await comp_tasks_repo.list_computational_tasks(computation.project_id)
     )
     if utils.is_pipeline_running(pipeline_state):
         raise HTTPException(
-            status_code=status.HTTP_403_FORBIDDEN,
+            status_code=status.HTTP_409_CONFLICT,
```

Comment:
Am I experiencing a deja vu 😕. I thought I fixed exactly this the other day?

Reply:
@bisgaard-itis our approach is still not very robust and it is error-prone. I have some proposals ... we will follow up.